uri/exid-types/RIN.yaml

```diff
@@ -38,4 +40,12 @@ specification:
         2 EXID 123abc
         3 TYPE https://gedcom.io/terms/v7/RIN#XYZ
 
+contact: GEDCOM@familysearch.org
+
+change controller: FamilySearch
+
```

Comment:
```suggestion
```

I think RINs are by definition specific to many programs, not specific to FamilySearch, I think the change controller should be the GEDCOM Steering Committee, not FamilySearch per se.